packages/desktop-client/src/components/autocomplete/NewPayeeAutocomplete.js

```diff
@@ -85,21 +85,21 @@ export default function PayeeAutocomplete({
           ? allOptions.filter(item => value.includes(item.value))
           : allOptions.find(item => item.value === value)
       }
-      isValidNewOption={input => input && !focusTransferPayees}
+      isValidNewOption={input => {
+        if (focusTransferPayees || !input) {
+          return false;
+        }
+
+        const lowercaseInput = input.toLowerCase();
+        const hasExistingOption = !!allOptions.find(
```

Comment:
I think `some` might be slightly more appropriate here

```suggestion
        const hasExistingOption = allOptions.some(
```